src/adsb_actions/adsbactions.py

```diff
@@ -27,6 +27,9 @@
 from .stats import Stats
 from .location import Location
 
+# Prometheus exporter
+from prometheus_client import start_http_server, Gauge
```

Comment:
all of this is unused now, right?  and all the Gauge declarations elsewhere in this PR, they are not incremented anymore?